src/views/core/members/UpdateProfile.vue

```diff
@@ -76,13 +76,28 @@
           </div>
         </div>
       </div>
+
+      <template v-if="validationErrors.privacy">
```

Comment:
also there's a commented form for editing user above, you can (and it'd be better to) add it there, so in case there's more stuff in the user profile filled in the wrong way, you can update it all at once, then you won't need savePrivacyConsent() at all and can use saveUser() for all of it

Reply:
Did you test the saveUser() function? It doesn't seem to redirect me to a different page after I've updated it

Reply:
As I gathered, we should check the validations again when a user submitted things, as now the router.push does not work since it still thinks some things need to be fixed. When you reload manually, there are no validation errors to fix anymore.